spec/Section 7 -- Response.md

```diff
@@ -135,14 +145,103 @@ locations, where each location is a map with the keys `line` and `column`, both
 positive numbers starting from `1` which describe the beginning of an
 associated syntax element.
 
+If an error can be associated to a particular field in the GraphQL result, it
+should contain an entry with the key `path` that details the path of the
+response field which experienced the error. This allows clients to identify
+whether a `null` result is intentional or caused by a runtime error.
+
+This field should be a list of path segments starting at the root of the
+response and ending with the field associated with the error. Path segments
+that represent object fields should be strings, and path segments that
+represent array indices should be 0-indexed integers. If the error happens
+in an aliased field, the path to the error should use the aliased name, since
+it represents a path in the response, not in the query.
```

Comment:
Yes. Also, more differences between query and response:

1. Arrays - those don't show up in the query
2. Arguments/aliases - these are reduced to exactly one field in the response

I don't think the error path has too much to do with the schema.